SupportYourLocals.Data/XMLData.cs

```diff
@@ -0,0 +1,164 @@
+﻿using MapControl;
+using System;
+using System.Collections.Generic;
+using System.IO;
+using System.Linq;
+using System.Security.Cryptography;
+using System.Text;
+using System.Xml;
+using System.Xml.Linq;
+
+namespace SupportYourLocals.Data
+{
+    public class XMLData : IDataStorage
+    {
+        private string filePath = @"./LocalSellersData.xml";
+        private string fileName = "LocalSellersData.xml";
+        XDocument doc = new XDocument();
+        XmlDocument document = new XmlDocument();
+        public XMLData()
```

Comment:
I'm not sure if this is the proper way to open a file in a constructor

Reply:
You should first try to see if a file exists, and only then try to load it. As it is now, the very first XDocument.Load throws and exception if you haven't manually created a file.

Reply:
Now it tries to find a file, creates it if doesn't exist and then loads it